api/admin/routes.py

```diff
@@ -393,22 +387,6 @@ def collection_library_registrations():
     )
 
 
-@app.route("/admin/admin_auth_services", methods=["GET", "POST"])
```

Comment:
This will require update to the admin UI, we will need to remove the Admin Auth from the settings so users on the CM cannot (try) to access these endpoints.

Reply:
https://github.com/ThePalaceProject/circulation-admin/pull/75